ods_ci/run_robot_test.sh

```diff
@@ -314,7 +314,13 @@ if command -v yq &> /dev/null
 fi
 
 if [[ ${SKIP_INSTALL} -eq 0 ]]; then
-  poetry install
+  # use a pre-created virtual environment if it exists
+  virtenv="${HOME}/.local/ods-ci/.venv"
+  if [[ -d "${virtenv}" ]]; then
+    poetry config --local virtualenvs.in-project true
+    ln --symbolic "${virtenv}" ./.venv
+  fi
+  poetry --no-interaction install --sync
 fi
 # shellcheck disable=SC1091
 source "$(poetry env info --path)/bin/activate"
```

Comment:
what about this line? wouldn't it try to use a different venv?

Reply:
It works fine, as intended. The config change I added above applies to all subsequent poetry executions from the ods-ci directory in Jenkins workspace.

Poetry keeps using the correct virtenv even in post build actions. The new PR in private Jenkins gitlab repository from @noam newly runs rebot through `poetry run rebot` and all is well.

Reply:
here's the internal MR for jenkins from @manosnoam ods/jenkins/-/merge_requests/625